tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -119,6 +131,15 @@ iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_waterm
     conditions:
       - "platform in ['x86_64-cel_e1031-r0']"
 
+#######################################
+#####           MACSEC            #####
+#######################################
+macsec/test_macsec.py:
```

Comment:
Order issue of `macsec` is already addressed in PR #5446.

Reply:
@wangxin rebased on master branch and commit it again.